### PR TITLE
feat: Generate coam app with NX according to #1755

### PR DESCRIPTION
**Pull request by gdalcengio**
#1755

### CODE REVIEW

File: bciers/package.json [hunk 1]
```diff
@@ -23,7 +23,9 @@
     "reg1:coverage": "nx run registration1:test --coverage",
     "reg1:test": "nx run registration1:test",
     "report": "nx dev reporting  -p 5000",
-    "report:build": "nx build reporting"
+    "report:build": "nx build reporting",
+    "coam": "nx dev coam",
+    "coam:build": "nx build coam"
```
Comment (marcellmueller) [inline bciers/package.json:28]: Can you please add `coam:test` command?

File: bciers/apps/coam/app/layout.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+export const metadata = {
```
Comment (marcellmueller) [inline bciers/apps/coam/app/layout.tsx:1]: Hey do you want to replace this entire file with the reusable layout that Shon recently added? Can just copy the layout from `/bciers/apps/dashboard/app/layout.tsx`.

That way this app will look like a bc gov app :)

File: bciers/apps/coam/app/page.tsx [hunk 3]
```diff
@@ -0,0 +1,467 @@
+export default function Index() {
+  /*
+   * Replace the elements below with your own.
+   *
+   * Note: The corresponding styles are in the ./index.css file.
+   */
+  return (
+    <div>
+      <div className="wrapper">
```
Comment (marcellmueller) [inline bciers/apps/coam/app/page.tsx:9]: Maybe remove all this Nx boilerplate HTML and just put something generic like:

`<h1>COAM</h1>` or whatever placeholder you think is best!

**Review by marcellmueller — COMMENTED** [review]
Awesome, thanks for the update! Just a couple minor things and this is good to go 🙂

**Comment by gdalcengio** [comment]
> Awesome, thanks for the update! Just a couple minor things and this is good to go 🙂

Updated, thanks for the feedback!

**Review by marcellmueller — APPROVED** [review]
Nice work, thanks for the updates!! 🥇 

I think there might be a few eslint errors to fix to pass the pre-commit check in CI but this is good other than that 🙂 

Let me know if you want a walkthrough of all the CI checks sometime soon!

**Pull request merged**